data/korath/korath outfits.txt

```diff
@@ -178,6 +206,292 @@ outfit "Control Transceiver"
+	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that may nearly melt the thruster."
+
+outfit "Afterburner (Comet Class)"
+	plural "Afterburners (Comet Class)"
+	category "Engines"
+	"cost" 299000
+	thumbnail "outfit/small korath afterburner"
+	"mass" 19
+	"outfit space" -19
+	"engine capacity" -19
+	"afterburner thrust" 78
+	"afterburner energy" 2.25
+	"afterburner heat" 22.75
+	"afterburner fuel" 0.1
+	"afterburner effect" "korath afterburner small"
+	description "Adding these afterburners to an interceptor allows it to reach absurd speeds; however, the heat produced ensures that running them for any substantial period of time will require some serious cooling."
+	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+
+outfit "Afterburner (Lunar Class)"
+	plural "Afterburners (Lunar Class)"
+	category "Engines"
+	"cost" 647000
+	thumbnail "outfit/medium korath afterburner"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"afterburner thrust" 148.8
+	"afterburner energy" 3.9
+	"afterburner heat" 41.65
+	"afterburner fuel" 0.18
+	"afterburner effect" "korath afterburner medium"
+	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, the prospect of installing one brings to mind many safety concerns."
+	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+
+outfit "Afterburner (Planetary Class)"
+	plural "Afterburners (Planetary Class)"
+	category "Engines"
+	"cost" 1459000
+	thumbnail "outfit/large korath afterburner"
+	"mass" 55
+	"outfit space" -55
+	"engine capacity" -55
+	"afterburner thrust" 288
+	"afterburner energy" 7.1
+	"afterburner heat" 79.45
+	"afterburner fuel" 0.32
+	"afterburner effect" "korath afterburner large"
+	"heat capacity" 500
+	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."
+	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+
+outfit "Afterburner (Stellar Class)"
+	plural "Afterburners (Stellar Class)"
+	category "Engines"
+	"cost" 3280000
+	thumbnail "outfit/huge korath afterburner"
+	"mass" 94
+	"outfit space" -94
+	"engine capacity" -94
+	"afterburner thrust" 552
+	"afterburner energy" 12.65
+	"afterburner heat" 148
+	"afterburner fuel" 0.58
+	"afterburner effect" "korath afterburner huge"
+	"heat capacity" 900
```

Comment:
```suggestion
	"heat capacity" 62
```

Reply:
Have these new values been tested? They look too low to be meaningful.

Reply:
These two I haven't extensively yet, they're just bumping it down to match the cooling in the same proportion as the LSC, so these specific numbers are essentially just my best starting placeholder for "this needs adjusted."

Because after playing around with the LSC adjustments to settle on those numbers, the original capacity values for these two are definitely -far, far- too high.

I'd myself also still like to see these both adjusted to have higher energy but lower heat usage than would be in line with the other Korath afterburners because of the powered cooling on them instead of them actually granting general extra cooling when used, and any decision on that will probably influence exactly how much heat capacity ends up being appropriate anyway.

Reply:
I think this value is very low. It would effectively provide a ship with a heat dissipation value of 0.5 with 186 cooling. A Mereti ship might get 350-400 effective cooling from this. That's not really a lot.

100 would be 300 cooling at 0.5 hear dissipation

Maybe around 150 for this one.